examples/SpringHibernateExample/README.md

```diff
@@ -0,0 +1,180 @@
+
+## Step 1: Create a Gradle Project
+Create a Gradle project with the following project hierarchy:
+
+```
+├───.mvn
+│   └───wrapper
+├───pom.xml
+└───src
+    ├───main
+        ├───java
+        │   └───example
+        |       ├───mvp
+        |           ├───MVP.java
+        |           └───MVPRepository.java
+        │       └───SpringHibernateExampleApplication.java
+        └───resources
+                └───application.yml
+```
+
+> Note: this sample code assumes the target database contains a table named MVP that can be generated using the SQL queries provided in `src/main/resources/mvp.sql`.
+
+`SpringHibernateExampleApplication.java` contains the following the code:
+
+```java
+@SpringBootApplication
+public class SpringHibernateExampleApplication implements CommandLineRunner {
+  private final Logger LOGGER = LoggerFactory.getLogger(this.getClass());
+
+  @Autowired
+  MVPRepository repository;
+
+  @Override
+  public void run(String... args) {
+    LOGGER.info("Mvp -> {}", repository.findAll());
+  }
+
+  public static void main(String[] args) {
+    SpringApplication.run(SpringHibernateExampleApplication.class, args);
+  }
+}
+```
+
+`MVP.java` contains the following code:
+```java
+package example.mvp;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+@Entity
+public class MVP {
+
+  @Id
+  @GeneratedValue
+  private int mvpId;
+
+  private int status;
+
+  public MVP() {
+    super();
+  }
+
+  public MVP(int id, int status) {
+    super();
+    this.mvpId = id;
+    this.status = status;
+  }
+
+  public MVP(int status) {
+    super();
+    this.status = status;
+  }
+
+  public int getMvpId() {
+    return mvpId;
+  }
+
+  public void setMvpId(int id) {
+    this.mvpId = id;
+  }
+
+  public int getStatus() {
+    return status;
+  }
+
+  public void setStatus(int name) {
+    this.status = name;
+  }
+
+  @Override
+  public String toString() {
+    return String.format("Mvp [id=%s, status=%s]", mvpId, status);
+  }
+}
+```
+Lastly, `MVPRepository.java` contains the following code:
+```java
+package example.mvp;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface MVPRepository extends JpaRepository<MVP, Integer> {
+
+}
+```
+
+## Step 2: Add the required Gradle Dependencies
+In your `build.gradle.kts`, add the following dependencies.
+
+```
+dependencies {
+   implementation("org.springframework.boot:spring-boot-starter-jdbc")
+   implementation("org.springframework.boot:spring-boot-starter-web")
+   implementation("org.postgresql:postgresql")
+   implementation("software.amazon.jdbc:aws-advanced-jdbc-wrapper")
+}
+```
+
+## Step 3: Configure Spring and Hibernate
+Configure Spring to use the AWS Advanced JDBC Driver as the default datasource.
+
+1. In the `application.yml`, add a new datasource for Spring:
+    ```yaml
+    datasource:
+        url: jdbc:aws-wrapper:postgresql://cluster-endpoint:5432/db
```

Comment:
To keep things consistent, can we use the sample values for endpoints and credentials that we have in the other example files?